docs/pages/access-controls/guides/impersonation.mdx

```diff
@@ -133,26 +142,33 @@ $ ssh -J jenkins@teleport.localhost:3023 -p 3022 jenkins@127.0.0.1
   about impersonator - `alice`.
 </Admonition>
 
-## Step 3/3. Dynamic impersonation
+## Advanced
+
+### Impersonation Rules
 
 To prevent unintended consequences, Teleport defines the following impersonation rules:
 
-- Even though Alice's `max_session_ttl` is 10 hours, she can issue a cert with a longer TTL of 240 hours,
-because `jenkins` role allows it. A certificate's TTL issued using impersonation is limited to the `max_session_ttl` of the roles being impersonated.
-- Even in case if `jenkins` role could impersonate some other roles, Alice would not be able
-to use this permission. Teleport prevents impersonating users recursively.
-- Alice can get a new `jenkins` certificate with the same TTL, but metadata updated, for example
-pointing to a different Teleport leaf cluster.
-Teleport allows impersonated users to renew their certificates with the reduced scope of the certificate.
+- Even though Alice's `max_session_ttl` is 10 hours, she can issue a cert with a
```

Comment:
@klizhentas could use your review here. I don't fully understand what this means. Allowing Alice to issue a cert with a *longer* TTL does feel like it is _limiting_ the TTL, it's actually extending it. Is this correct?

Reply:
yes, better would say that certificate's TTL is extended to the `max_session_ttl` of the roles that are being impersonated.

Reply:
Ok understood. I will update this section.